bin/check

```diff
@@ -6,6 +6,6 @@
 #> bin/check
 
 bin/check_govet
-bin/check_golint
+bin/builder_run bin/check_golint
```

Comment:
why do we need to run `golint` through the builder docker container?

Reply:
It seems that something happened with script - I don't know. Running inside docker was invented to keep script usable on local machines :| and now it seems we need to use docker even locally :(

Reply:
hm, it works for me locally. Could you give more details what is not working for you?

Reply:
Can you comment on whats not working without `builder_run` ? We should not increase our build time if its not needed.

Reply:
for me bin/check_golint is not showing lint warnings